.gitignore

```diff
@@ -12,3 +12,6 @@ pom.xml.asc
 /node_modules
 /_book
 figwheel_server.log
+
+/.m2
+/.ash_history
```

Comment:
How do you end up with these in the repo in the first place?

Reply:
I tend to dev inside docker with a command like
```
docker run -it -w /root -v "$(pwd):/root" -p 9001:9001 clojure:lein-alpine sh
```
Since it's mounted as a volume, when I run the scripts in [the guide](https://metosin.github.io/reitit/development.html) or `lein repl`, these flow back.

Reply:
As those aren't really specific to this project, maybe you should ignore those on your global git config?

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
You're right, gonna revert